cmd/daemonSetCheck/Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM golang:1.13 AS builder
+FROM golang:1.13-alpine AS builder
+RUN apk upgrade
 ADD . /build
 WORKDIR /build/cmd/daemonSetCheck
+CMD CGO_ENABLED=0 go test ./...
```

Comment:
always run `go test` with `-v` in dockerfiles so you know what goes wrong in the build log.  Also, why are you using `CGO_ENABLED=0` here?

Reply:
cause i get this error if i don't enable cgo
```# runtime/cgo
exec: "gcc": executable file not found in $PATH```